### DISCUSSION TITLE
CF App Transfer Complete

### DISCUSSION THREAD

**Discussion by stephen-hunt** [nasa/cFS#729] [Q&A] [answered]
Question regarding the CF, cFS app. Is there any way to know within the app that a new transfer has completed (both upload and download)? Something that is available to outside users, like within the HK Message?

**Comment by skliper** [comment, marked as answer]
That's the intent behind the end of transmission packet:
https://github.com/nasa/CF/blob/ab17422f0959f4c1e5cf6a9a6b0537208deefc5a/config/default_cf_msgstruct.h#L53-L60

**Reply by stephen-hunt** [reply]
Awesome, thank you!
